style.css

```diff
@@ -1,36 +1,860 @@
+    grid-template-columns: repeat(2, 50%);
+    gap: 0.5em;
+    background-color: rgba(121, 121, 121, 0.3);
+    height: 90%;
+    margin-left: 2%;
+    margin-right: 2%;
+  }
+
+  .prog {
+    display: none;
+  }
+
+  .card-box:hover {
+    transform: scale(0.9);
+    border: solid orangered;
+    border-width: 1px;
+  }
+
+  .card-box p {
+    font-family: "Lato", sans-serif;
+    color: white;
+  }
+
+  .grid {
+    display: flex;
+    margin-left: 5%;
+  }
+
+  .lines {
+    border: solid orangered;
+    width: 30%;
+    margin-left: auto;
+    margin-right: auto;
+    border-width: 1.5px;
+  }
+
+  #head-p {
+    display: none;
+  }
+
+  .grid .p1 {
+    color: orangered;
+    font-family: "Lato", sans-serif;
+    font-size: 14px;
+    margin-left: 5%;
+  }
+
+  .grid .p2 {
+    font-size: 12px;
+    margin-left: 5%;
+  }
+
+  .grid h2 {
+    font-size: 20px;
+    margin-left: 5%;
+    padding-top: 10%;
+    font-weight: bolder;
+  }
+
+  .card-box h3 {
+    font-size: 120%;
+    color: rgb(255, 104, 50);
+    padding-top: 30px;
+    padding-left: 5%;
+  }
+
+  .program-box img {
+    padding-top: 10%;
+    max-width: 70%;
+    max-height: 70%;
+  }
+
+  .program-section {
+    position: relative;
+    background: url(Img/mainbg.webp);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    background-size: cover;
+  }
+
+  .speakers {
+    padding-top: 8%;
+  }
+
+  #speaker-title {
+    margin-left: 20%;
+  }
+
+  .join {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 20%;
+    padding-bottom: 10%;
+  }
+
+  .btn {
+    background-color: rgb(223, 34, 1);
+    padding: 16px;
+    text-decoration: none;
+    color: white;
+  }
+
+  .date h2 {
+    margin-top: 1px;
+  }
+
+  .contact {
+    margin-left: 18%;
+    margin-top: 5%;
+  }
+
+  .about {
+    height: 100vh;
+  }
+
+  .about-second {
+    display: flex;
+    flex-direction: column;
+    background-color: white;
+    margin-top: 15%;
+    padding-top: 10%;
+  }
+
+  .about-child {
+    width: 70%;
+    display: flex;
+    flex-direction: column;
+    margin-left: 12%;
+  }
+
+  #about-logo {
+    width: 90%;
+    border: solid grey;
+    border-width: 1px;
+    margin-left: 5%;
+  }
+
+  #the {
+    margin-left: 20%;
+  }
+
+  .past {
+    background-color: white;
+  }
+
+  .past-events {
+    display: flex;
+    flex-direction: column;
+    row-gap: 20px;
+    margin: 5%;
+  }
+
+  .past-events img {
+    width: 100%;
+  }
+
+  .par-flex {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    row-gap: 8%;
+    align-items: center;
+    color: rgb(117, 117, 117);
+    font-weight: bolder;
+    height: 50vh;
+  }
+
+  .f-par {
+    display: flex;
+    column-gap: 13%;
+    margin-left: -10%;
+  }
+
+  .f-par2 {
+    display: flex;
+    column-gap: 7%;
+  }
+
+  .partners {
+    background-color: rgb(53, 53, 53);
+  }
+
+  .footer {
+    display: flex;
+    background-color: white;
+  }
```

Comment:
- [ ] We are very close to finishing the project. According to the original design guideline, the desktop version of the about section should be dark and white in mobile version. Kindly consider applying this to match the original design ☺️.

Original design desktop😎 
![image](https://github.com/Zel-hub7/Capstone-project1/assets/99394374/6b17fb76-1b9f-400c-a077-33b466e76792)

Mobile
![image](https://github.com/Zel-hub7/Capstone-project1/assets/99394374/11aadd95-2ad8-45d2-bc96-a8322c779905)